DFC.ServiceTaxonomy.GraphSync/Orchestrators/ContentTypeOrchestrator.cs

```diff
@@ -74,8 +74,8 @@ public void SetDefaultsForGraphSyncPart(string contentTypeName)
                 _logger.LogError(e, "Failed setting data synchronisation defaults for {ContentType} content type.",
                     contentTypeName);
 
-                _notifier.Add(NotifyType.Error, new LocalizedHtmlString(nameof(GraphSyncContentDefinitionHandler),
-                    $"Failed setting data synchronisation defaults for {contentTypeName} content type."));
+                _notifier.AddAsync(NotifyType.Error, new LocalizedHtmlString(nameof(GraphSyncContentDefinitionHandler),
```

Comment:
Needs awaiting I imagine

Reply:
Also why ConfigureAwait here but not on the next few examples?

Reply:
Have amended the code to get the result